site/app/views/forum/ForumThreadView.php

```diff
@@ -251,54 +252,54 @@ function changeName(element, user, visible_username, anon){
 HTML;
 					$first = true;
 					$first_post_id = 1;
-					$order_array = array();
-					$reply_level_array = array();
-					foreach($posts as $post){
-						if($thread_id == -1) {
-							$thread_id = $post["thread_id"];
-						}
+						$order_array = array();
+						$reply_level_array = array();
+						foreach($posts as $post){
+							if($thread_id == -1) {
+								$thread_id = $post["thread_id"];
+							}
 
-						if($first){
-							$first= false;
-							$first_post_id = $post["id"];
-						}
-						if($post["parent_id"] > $first_post_id){
-							$place = array_search($post["parent_id"], $order_array);
-							$tmp_array = array($post["id"]);
-							$parent_reply_level = $reply_level_array[$place];
-							while($place && $place+1 < sizeof($reply_level_array) && $reply_level_array[$place+1] > $parent_reply_level){
-								$place++;
+							if($first){
+								$first= false;
+								$first_post_id = $post["id"];
 							}
-							array_splice($order_array, $place+1, 0, $tmp_array);
-							array_splice($reply_level_array, $place+1, 0, $parent_reply_level+1);
-						} else {
-							array_push($order_array, $post["id"]);
-							array_push($reply_level_array, 1);
+							if($post["parent_id"] > $first_post_id){
+								$place = array_search($post["parent_id"], $order_array);
+								$tmp_array = array($post["id"]);
+								$parent_reply_level = $reply_level_array[$place];
+								while($place && $place+1 < sizeof($reply_level_array) && $reply_level_array[$place+1] > $parent_reply_level){
+									$place++;
+								}
+								array_splice($order_array, $place+1, 0, $tmp_array);
+								array_splice($reply_level_array, $place+1, 0, $parent_reply_level+1);
+							} else {
+								array_push($order_array, $post["id"]);
+								array_push($reply_level_array, 1);
 
+							}
 						}
-					}
-					$i = 0;
-					$first = true;
-					foreach($order_array as $ordered_post){
-						foreach($posts as $post){
-							if($post["id"] == $ordered_post){
-								if($post["parent_id"] == $first_post_id) {
-									$reply_level = 1;	
-								} else {
-									$reply_level = $reply_level_array[$i];
+						$i = 0;
+						$first = true;
+						foreach($order_array as $ordered_post){
+							foreach($posts as $post){
+								if($post["id"] == $ordered_post){
+									if($post["parent_id"] == $first_post_id) {
+										$reply_level = 1;	
+									} else {
+										$reply_level = $reply_level_array[$i];
+									}
+									
+									$return .= $this->createPost($thread_id, $post, $function_date, $title_html, $first, $reply_level, $display_option);
+									break;
 								}
 								
-								$return .= $this->createPost($thread_id, $post, $function_date, $title_html, $first, $reply_level);
-								break;
 							}
-							
-						}
-						if($first){
-							$first= false;
+							if($first){
+								$first= false;
+							}
+							$i++;
 						}
-						$i++;
-					}
-
+					
```

Comment:
It looks like you just indented everything by one level unnecessarily? Please unindent the block back to match the indentation level of the stuff above it.